### ISSUE TITLE
Compliance/Penalty - Internal finish issuing Imposed penalty

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#82]
#### Description:

A CAS staff member needs to be able to complete the issuance of an imposed penalty to an operator on the 'Enter details & issue penalty tab'.

#### Acceptance Criteria:

As a CAS staff member, given that I clicked the continue button from the previous screen, when I open the Enter Details & Issue penalty page, I can fill in penalty amounts and issue the penalty by clicking the issue button.

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=23878-8317&t=e4Tuoo3a9HAnrEND-1)

- [ ] Page title: Enter Details & Issue Penalty
- [ ] Field name: Penalty amount/ Field type: numeric
- [ ] Field name: Issuance date/ Field type: date picker
- [ ] Field name: Due date/ Field type: date picker
- [ ] Field name: Reason for penalty/ Field-type: drop-down selection list
- [ ] Field name: Comments/ Field type: free-text box
- [ ] Back button (to previous tab)
- [ ] Issue button: This button will actually issue the penalty via API to eLicensing, and create an entry in the Imposed Penalty grid.

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by: #81 
Blocking #83 
Blocking #84 

#### Notes:

-
